go/service/config.go

```diff
@@ -581,19 +581,11 @@ func (h ConfigHandler) UpdateLastLoggedInAndServerConfig(
 		SessionType: libkb.APISessionTypeREQUIRED,
 	}
 	mctx := libkb.NewMetaContext(ctx, h.G())
-	resp, finisher, err := h.G().API.GetResp(mctx, arg)
-	defer finisher()
-	if err != nil {
-		return err
-	}
-	userFeatureusBytes, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return err
-	}
-	jw, err := jsonw.Unmarshal(userFeatureusBytes)
```

Comment:
cc @songgao, don't think we need to stream the response here, can just let the api library do the json unmarshal